components/form-control/README.md

```diff
@@ -0,0 +1,341 @@
+<!--Copyright (c) Coherent Labs AD. All rights reserved. Licensed under the MIT License. See License.txt in the project root for license information. -->
+The gameface-form-control is part of the Gameface custom components suite. As most of the components in this suite, it uses slots to allow dynamic content.
+
+# Installation
+
+`npm i coherent-gameface-form-control`
+
+# Usage
+
+The gameface-form-control component comes with UMD and CJS builds.
+
+## Usage with UMD modules:
+
+* import the components library:
+
+~~~~{.html}
+<script src="./node_modules/coherent-gameface-components/umd/components.production.min.js"></script>
+~~~~
+
+* import the gameface-form-control component:
+
+~~~~{.html}
+<script src="./node_modules/gameface-form-control/umd/gameface-form-control.production.min.js"></script>
+~~~~
+
+This is all! Load the file in Gameface to see the gameface-form-control.
```

Comment:
Shouldn't there be some html with usage like:

`<gameface-form-control></gameface-form-control>`

?

Reply:
Yes.. I generated readme md via cli so this example has been skipped ..